packages/utilities/src/ImperativeConfig.ts

```diff
@@ -301,4 +302,21 @@ export class ImperativeConfig {
             ImperativeConfig.instance.loadedConfig.profiles.forEach(profile => schemas[profile.type] = profile.schema);
         return schemas;
     }
+
+    /**
+     * Checks if obj complies with the { @link ICredentialManager } interface
+     * @throws ImperativeError if object is of type ICredentialManager but does not have required 'type' property
+     * @returns boolean representing whether it complies or not with the interface
+     */
+    public isCredentialManager(obj: any): obj is ICredentialManager {
+        if(typeof obj !== 'object' && obj !== null) {
```

Comment:
What if `obj is `undefined` ?

should we use `obj != null`  instead of strict inequality?